iosHackthonApp/Modules/Features/Login/Sources/Login/LoginView.swift

```diff
@@ -46,16 +53,21 @@ public struct LoginView: View {
 						Text(Strings.Login.email)
 						TextField(Strings.Login.emailPlaceholder, text: $email)
 							.modifier(GreyTextFieldStyle())
+							.autocapitalization(.none)
```

Comment:
please can we add
`.disableAutocorrection(true)`
to this also...my name isn't "Garett" 😭

Reply:
Done